src/SendGrid/SendGridClient.cs

```diff
@@ -50,8 +50,8 @@ public SendGridClient(HttpClient httpClient, string apiKey, string host = null,
         /// <param name="version">API version, override AddVersion to customize.</param>
         /// <param name="urlPath">Path to endpoint (e.g. /path/to/endpoint).</param>
         /// <returns>Interface to the Twilio SendGrid REST API.</returns>
-        public SendGridClient(string apiKey, string host = null, Dictionary<string, string> requestHeaders = null, string version = null, string urlPath = null)
-            : base(buildOptions(apiKey, host, requestHeaders, version, urlPath))
+        public SendGridClient(string apiKey, string? host = null, Dictionary<string, string>? requestHeaders = null, string? version = null, string? urlPath = null, bool httpErrorAsException = false)
```

Comment:
Think the addition of the last param should be refactored into a separate method since it's public and breaks existing linkages.

Reply:
Reverted the change since it's out of the scope of this PR